### ISSUE TITLE
Upon team lead completing a request, notify next requester

### ISSUE THREAD

**Issue by stevejgordon** [HTBox/allReady#1907]
On Installer marking previous request in itinerary delivered day of send "See you in a few" message

We don't currently have the ability for an installer to mark request as completed. We do have a requirement to add this in #1788 

Once completed, we can add the ability to optionally send this message as part of the closure of the previous request.

**Comment by mgmccarthy** [comment]
@stevejgordon, I'm trying to look into what it will take to implement this, but it's a bit of a maze between all the linked issues and the PR's...

I'm assuming here if  a team lead ends up sending the ChangeRequestStatusCommand with a status of Completed from the AllReady.Areas.Admin.Controllers.ItineraryController, that is when we'd want to do the lookup of the next requestor on the itinerary and let them know we're coming via "See you in a few" message

**Comment by stevejgordon** [comment]
@mgmccarthy Yep that's probably the best flow and what I had in the back of my mind.
